index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <header>
+        <div class="status-bar">
+            <p class="hour">9:41</p>
+            <div class="icons">
+                <div class="network">
+                    <img src="images/toolbar_icons/connection.png" alt="mobile connection icon">
+                </div>
+                <div class="wifi">
+                    <img src="images/toolbar_icons/wifi.png" alt="wifi icon">
+                </div>
+                <div class="battery">
+                    <img src="images/toolbar_icons/battery.png" alt="battery icon">
+                </div>
+            </div>
+        </div>
+        <div class="menu">
+            <h6 class="name">John Doe</h6>
+            <button type="button" aria-label="open close main menu" class="toolbar-icon">
+                <img src="images/toolbar_icons/menu.png" alt="">
+            </button>
```

Comment:
Please ensure that the hamburger button has a `nav` tag according to the requirement.